main.go

```diff
@@ -67,6 +68,9 @@ var (
 	globalConsoleDisplay bool
 	globalConnStats      []*ConnStats
 	log2                 *logrus.Logger
+	globalClientIP       string
+	globalForwardedHost  string
```

Comment:
You seem to be not using these global values ?

Reply:
L#660 and L#676 using them, right?